src/sentry/static/sentry/app/utils/formatters.tsx

```diff
@@ -1,8 +1,9 @@
 import {get} from 'lodash';
 
 import {t} from 'app/locale';
+import {User} from 'app/types';
 
-export function userDisplayName(user) {
+export function userDisplayName(user: User): string {
```

Comment:
One line 7 this function uses `get(user, 'name', ...)` implying that email is an optional property. Should the type reflect that?

Reply:
Good point. Looking back at https://github.com/getsentry/sentry/pull/13707, I see why this is the case now. I think I'll add a CommitAuthor type and type user as `User | CommitAuthor` so that we can keep the stricter typing for users.

Reply:
Updated, thanks for catching that!